### PR TITLE
Clarify in `Slide In` dialog that a new branch can be created

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties [hunk 1]
```diff
@@ -135,7 +135,7 @@ action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.ok-button=Slide In
 action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.title=Slide In Branch Below
 action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.no-branch-selected=A branch should be selected.
 action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.error.slide-in-under-itself=Cannot slide in the branch under itself.
-action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.placeholder=Specify branch
+action.GitMachete.BaseSlideInBelowAction.dialog.slide-in.placeholder=Specify a new or an existing branch
```
Comment (PawelLipski) [inline frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties:138]: Here's how it looks like now...

image

What else can be changed... `Slide In New or Existing Branch` in the dialog title? although that would be probably too much

Comment (mkondratek) [reply frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties:138]: I think it's now more precise and looks much better - should be enough. 

However, if we want to verify whether the changed title is better you could provide a screenshot with the changed version.

**Review by mkondratek — APPROVED** [review]

File: frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties (hunk 1 above)
Comment (PawelLipski) [reply frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties:138]: image

WDYT?

File: CHANGE-NOTES.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 - Improved Override Fork Point dialog: inapplicable options are hidden, placeholders are no longer displayed.
 - Fixed an error with a too long GitHub URI when reporting exceptions in the plugin.
 - Fixed a too short traverse info dialog.
+- Clarified Slide In dialog: a branch to slide in can be created on the spot
```
Comment (mkondratek) [inline CHANGE-NOTES.md:9]: I would try to stick to the words Added/Fixed/Improved (Improved in this case ofc) - no strong feelings

File: frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties (hunk 1 above)
Comment (mkondratek) [reply frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties:138]: +1 to the changed title

**Pull request merged**